tree_random_test.go

```diff
@@ -28,9 +30,12 @@ func TestRandomOperations(t *testing.T) {
 		957382170,
 	}
 
-	for _, seed := range seeds {
-		seed := seed
+	for i, seed := range seeds {
+		i, seed := i, seed
 		t.Run(fmt.Sprintf("Seed %v", seed), func(t *testing.T) {
+			if testing.Short() && i >= 2 {
```

Comment:
😄

Reply:
Hm?

Reply:
Just a smile for going with shortening the amount of test cases.